index.js

```diff
@@ -46,7 +46,7 @@ interface Answer {
     label: String
     type: AnswerType!
     mandatory: Boolean
-    questionPageId: Int!
+    questionPage: QuestionPage
```

Comment:
Can we call this `page`? When we have multiple types of pages, each with possible child objects, it'd be better to be consistent and use `page` in all children (the "question" part is implied by the fact this is an answer)